address.py

```diff
@@ -36,36 +30,8 @@ class Address(metaclass=PoolMeta):
         'get_address_contact_mechanisms')
 
     def get_address_contact_mechanism(self, name):
-        pool = Pool()
```

Comment:
Since 38e7ade get_address_contact_mechanism() method check contact mechanisms that has write_date/create_date.

But source code from party module, has a new method, contact_mechanism_get() that return first record in contact mechanims. This new method is used in some modules that stock_package_shipping_ups, ....

An example from this module:

```
mobile = shipping_to_address.contact_mechanism_get('mobile')
```

For this reason, to get "best value" is not from create/write date. An example:

1. create first contact mechanism: 1@demo com (without the dot)
2. create second contact mechanism: 2@demo.com
3. FIX firts email to 1@demo.com

Current code return 1@demo.com but maybe is better return 2@demo.com

We add a new field is_default and user do control witch email is the default.